src/index.html

```diff
@@ -1,12 +1,291 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__content">
+    <header class="header">
+      <a href="#" class="header__link">
+        <img
+          class="header__logo"
+          src="./images/logo.png"
+          alt="Moyo logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover">Laptops & Computers</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main-content">
+      <div class="card" data-qa="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+          class="card__picture"
+        >
+        <h2 class="card__product-name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__product-code">Product code: 195434</p>
+        <div class="card__rating-panel">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__review-counter">Reviews: 5</p>
+        </div>
+        <div class="card__price-block">
```

Comment:
![image](https://user-images.githubusercontent.com/91411692/222673221-e8e3341f-49f2-408f-b188-f57752416385.png)
```suggestion
        >
        <h2 class="card__product-name">
          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
        </h2>
        
        <p class="card__product-code">Product code: 195434</p>
        
        <div class="card__rating-panel">
          <div class="stars stars--4">
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
          </div>
          
          <p class="card__review-counter">Reviews: 5</p>
        </div>
        
        <div class="card__price-block">
```

Reply:
Fixed